app/src/main/java/edu/uc/zhukv/droneradarmap/main/CurrentWeatherFragment.kt

```diff
@@ -0,0 +1,5 @@
+package edu.uc.zhukv.droneradarmap.main
+
+class CurrentWeatherFragment {
+    //TODO: Show Local Forecast
```

Comment:
1) Only features that meet the Definition of Done should be merged to Master.
2) Unused code is clutter, which increases technical debt.